src/platform/src/app/(pages)/(layout-1)/settings/page.jsx

```
@@ -0,0 +1,104 @@
+import { useEffect, useState } from 'react';
+import { useSession } from 'next-auth/react';
+import { getAssignedGroupMembers } from '@/core/apis/Account';
+import Profile from './Tabs/Profile';
+import OrganizationProfile from './Tabs/OrganizationProfile';
+import ErrorBoundary from '@/components/ErrorBoundary';
+import { setChartTab } from '@/lib/store/services/charts/ChartSlice';
+import API from './Tabs/API';
+
+export const checkAccess = (requiredPermission, rolePermissions) => {
+  const permissions =
+    rolePermissions && rolePermissions.map((item) => item.permission);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)